fastpass/src/main/scala/scala/meta/internal/fastpass/generic/CreateOptions.scala

```diff
@@ -21,9 +21,9 @@ case class CreateOptions(
     )
     name: Option[String] = None,
     @Description(
-      "Whether to use Bazel for project import. Inferred from target specs."
+      "Whether to use Pants for project import. Inferred from target specs."
     )
-    bazel: Boolean = false,
+    pants: Boolean = false,
```

Comment:
Won't this break the scripts that call Fastpass with `--bazel`? Would it be possible instead to switch `--bazel` to `true` and add `--pants` which defaults to false? Settings both would be an error.

Reply:
Restored bazel option and changed pants option to forcePants. Restored old pants option from pantsPath